cmd/proxy/main.go

```diff
@@ -44,6 +44,20 @@ func main() {
 
 	logger := athenslog.New(conf.CloudRuntime, logLvl)
 
+	// logrus.Writer() spawns a go routine, so instead of doing that each time
+	// we need to make logrus look like a (standard) log.Logger (and then Close()ing it),
+	// we can do it once in main and have the global log.Logger use it.
```

Comment:
```suggestion
	// Use the logrus writer as the standard logger's output in order to avoid starting goroutines whenever logrus.Writer() is called.
```

WDYT of this simplification of the comment?

Reply:
I think it's fair to say that this was too long a comment for code and better off in the PR/commit explanation. I went with something even shorter in 6eea0a6.